test/unit/3dtileslayerprocess.js

```diff
@@ -1,34 +1,68 @@
 import assert from 'assert';
+import fs from 'fs';
 import C3DTilesLayer from 'Layer/C3DTilesLayer';
 import C3DTilesSource from 'Source/C3DTilesSource';
 import View from 'Core/View';
 import GlobeView from 'Core/Prefab/GlobeView';
 import { HttpsProxyAgent } from 'https-proxy-agent';
 import Coordinates from 'Core/Geographic/Coordinates';
+import sinon from 'sinon';
+import Fetcher from 'Provider/Fetcher';
 import Renderer from './bootstrap';
 
+import tilesetDiscretLOD from '../data/3dTiles/tilesetDiscretLOD.json';
+
+const urlDragonLow = 'https://raw.githubusercontent.com/CesiumGS/3d-tiles-samples/master/1.0/TilesetWithDiscreteLOD/dragon_low.b3dm';
+const urlDragonMedium = 'https://raw.githubusercontent.com/CesiumGS/3d-tiles-samples/master/1.0/TilesetWithDiscreteLOD/dragon_medium.b3dm';
+
+const dragonLow = fs.readFileSync('./test/data/3dTiles/dragon_low.b3dm');
+const dragonMedium = fs.readFileSync('./test/data/3dTiles/dragon_medium.b3dm');
```

Comment:
Use a relative path.

```suggestion
const dragonMedium = fs.readFileSync('../data/3dTiles/dragon_medium.b3dm');
```

Reply:
same